### ISSUE TITLE
SSL / HTTPS

### ISSUE THREAD

**Issue by maxkfranz** [GeneMANIA/genemania#50]
http://www.startssl.com/

- Needed for Dropbox integration
- Would keep queries more private

**Comment by tobiasg82** [comment]
I agree that would be great. Please feel free to e-mail me at tobi.guennel@precisionformedicine.com if any support is needed to get this accomplished in case resources (people or monetary) are needed.
